README.md

```diff
@@ -12,23 +12,25 @@ and TCFv2 to everyone else.
 
 ## Table of Contents
 
+<!-- this TOC is auto generated – don't edit it :) -->
+
 <!-- toc -->
 
--   [Installation](#installation)
```

Comment:
this shouldn't be happening – its auto generated, i'll look into it in a seperate PR

Reply:
I think it was changed when I edited them, too. Maybe an odd prettier setting?